src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/QueueBindingProvider.cs

```diff
@@ -64,9 +64,13 @@ internal class QueueBindingProvider
             var bindingFactory = new BindingFactory(nameResolver, converterManager);
 
             var bindAsyncCollector = bindingFactory.BindToAsyncCollector<QueueAttribute, IStorageQueueMessage>(BuildFromQueueAttribute, ToWriteParameterDescriptorForCollector, CollectAttributeInfo);
```

Comment:
I like how you removed the BuildParameterDescriptor/PostResolveHook from the public API below. Can you do the same for the BindToAsyncCollector?